apps/builder/app/builder/shared/tree/index.tsx

```diff
@@ -59,21 +65,39 @@ export const InstanceTree = (
     [instances]
   );
 
+  const updateInstanceLabel = useCallback((instanceId: string, val: string) => {
```

Comment:
please use full names
```suggestion
  const updateInstanceLabel = useCallback((instanceId: string, value: string) => {
```